### PR TITLE
Updated to support cloud build

### PR DESCRIPTION
**Pull request by railroadmanuk**
* Updated README file
* Added cloudbuild.yaml file for Cloud Build pipelines
* Added `-e` flag to rules.py to enable passing credential info via environment variable

### CODE REVIEW

File: cicd/README.md [hunk 1]
```diff
@@ -96,3 +99,32 @@ The resulting string can then be pasted into the Secrets UI in GitHub.
 
 Now whenever a change is written to the repository the contents of the passed rules folder will be checked and updated/uploaded
 on the Chronicle instance.
+
+## Using in a Google Cloud Build pipeline
+
+The [`cloudbuild.yaml`](./cloudbuild/cloudbuild.yml) file located in the
```
Comment (rarsan) [inline cicd/README.md:105]: Typo in file url (yaml vs yml)

File: cicd/README.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ optional arguments:
                         path to credentials file
   -i CREDENTIALS_INFO, --credentials_info CREDENTIALS_INFO
                         service account credentials info (alternative to file)
+  -e CREDENTIALS_ENV, --credentials_env CREDENTIALS_ENV
+                        service account credentials info stored in environment
+                        variable
```
Comment (rarsan) [inline cicd/README.md:42]: do we need to specify what happens if multiple creds are set? e.g. file and env are set, or info and env, which takes precedence?

**Review by rarsan — CHANGES_REQUESTED** [review]
LGTM, two minor requested changes below. Thx for adding this!

File: cicd/README.md (hunk 1 above)
Comment (railroadmanuk) [reply cicd/README.md:108]: Resolved in last push

File: cicd/README.md (hunk 2 above)
Comment (railroadmanuk) [reply cicd/README.md:42]: Resolved in last push

**Pull request merged**